packages/xstate-test/src/TestModel.ts

```diff
@@ -283,28 +283,21 @@ export class TestModel<TState, TEvent extends EventObject> {
         testPathResult.steps.push(testStepResult);
 
         try {
-          await this.testState(params, step.state, options);
+          await this.testTransition(params, step);
         } catch (err: any) {
-          testStepResult.state.error = err;
+          testStepResult.event.error = err;
 
           throw err;
         }
 
         try {
-          await this.testTransition(params, step);
+          await this.testState(params, step.state, options);
         } catch (err: any) {
-          testStepResult.event.error = err;
+          testStepResult.state.error = err;
 
           throw err;
         }
       }
-
-      try {
```

Comment:
shouldn't similar changes be made to `testPathSync`? those are 2 sibling methods that have almost the same implementation

Reply:
Going to remove that one

Reply:
Will remove later, this is done